test/e2e/density.go

```diff
@@ -359,9 +196,9 @@ var _ = Describe("Density", func() {
 					for i := 0; i < itArg.rcsPerThread; i++ {
 						name := "my-short-lived-pod" + string(util.NewUUID())
 						n := itArg.podsPerMinion * minionCount
-						RunRC(c, name, ns, "gcr.io/google_containers/pause:go", n)
+						expectNoError(RunRC(c, name, ns, "gcr.io/google_containers/pause:go", n))
 						podsLaunched += n
-						glog.Info("Launched %v pods so far...", podsLaunched)
+						Logf("Launched %v pods so far...", podsLaunched)
```

Comment:
Correct.  Ginkgo allows parallelizing tests and other features and using its logging facility ensures the logs stay with the test and get rolled up in reports generated from ginkgo logs.